boards/msba2-common/Makefile

```diff
@@ -1,4 +1,4 @@
-MODULE = board
+MODULE = msba2-common
```

Comment:
Side note: @haukepetersen changed this not a long time ago to `board`, because it would make it easier for his size copmarison tool